Experimental/conda/environment.yml

```diff
@@ -0,0 +1,11 @@
+name: fab
+dependencies:
+  - python=3.8
+  - pip=21.2.4
+  - flake8
+  - pytest
+  - pytest-mock
+  - pytest-cov
+  - libclang
+  - python-clang
+  - mypy
```

Comment:
I wonder whether we should be explicit about the versions of everything?  (or set minimum versions as in the other comment).  I guess specifying the list without qualifiers just means get the most recent version of each package?

Reply:
Yes, it'll get the latest version available to conda, including future bug fixes and potentially breaking changes for major versions. I don't know how we'd decide on minimum versions, other than pinning to whatever we're using right now, which seems arbitrary and might be newer than potential external collaborators have available. I suggest we leave it, especially as it's just copying the wiki command.